tutorials/36_Building_Pipelines_With_Conditional_Routing.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<p>Do we nees ro have 2 prompt templates? I think we could make it so that the one promopt template can have a {% if documents_from_web %} block which gets filled in if there's a route from <code>go_to_websearch</code>  but maybe I'm off here..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
It's totally doable as an alternative solution where we create a loop in the pipeline. I didn't want to explain the loop in this tutorial but rather wanted to explain branches